src-tauri/plugins/tauri-plugin-native-utils/android/src/main/java/com/dashbeam/plugin/native_utils/NativeUtils.kt

```
@@ -137,6 +137,17 @@ class NativeUtils(private val activity: Activity) : Plugin(activity) {
         }
     }
 
+    @Command
+    fun get_window_insets(invoke: Invoke) {
+        activity.runOnUiThread {
+            try {
+                invoke.resolveObject(activity.readWindowInsets())
+            } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)